shipyard-controller/handler/shipyardcontroller_utils.go

```diff
@@ -49,48 +48,6 @@ func GetStageFromShipyard(stageName string, shipyard *keptnv2.Shipyard) *keptnv2
 	return nil
 }
 
-func GetNextTaskOfSequence(taskSequence *keptnv2.Sequence, previousTask *models.TaskExecution, eventScope *models.EventScope, eventHistory []interface{}) *models.Task {
```

Comment:
this function is now offered as a method of the `SequenceExecution` model. This should help to separate the concerns